src/Magnum/DartIntegration/Test/CMakeLists.txt

```diff
@@ -42,9 +43,8 @@ endif()
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
                ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
 
-corrade_add_test(DartIntegrationDartSkeletonTest DartSkeletonTest.cpp LIBRARIES MagnumDartIntegration ${DART_LIBRARIES})
-target_include_directories(DartIntegrationDartSkeletonTest PUBLIC ${DART_INCLUDE_DIRS})
+corrade_add_test(DartIntegrationWorldGLTest WorldGLTest.cpp LIBRARIES Magnum::OpenGLTester MagnumDartIntegration ${DART_LIBRARIES})
```

Comment:
Here we need to link to DART libs, because for the test we require more (i.e., either `utils-urdf` or `io-urdf`) libs from DART than the rest of the integration and as such cannot link properly..